packages/react-components/src/templates/WelcomePage.tsx

```diff
@@ -50,7 +50,7 @@ const containerStyles = css({
 });
 
 const headerStyles = css({
-  flexGrow: 9999,
+  flexGrow: 1,
```

Comment:
Thank you

Reply:
lol, I kinda just moved the hack

Reply:
but it's a fairly common CSS hack :shrug: